CMakeLists.txt

```
@@ -90,6 +100,7 @@ install(
   src/ego-gdb
   src/ego-go
   ${CMAKE_BINARY_DIR}/ego
+  ${CMAKE_BINARY_DIR}/ego-bundle
   DESTINATION ${CMAKE_INSTALL_BINDIR})
```

```suggestion
  DESTINATION ${CMAKE_INSTALL_BINDIR})
install(FILES ${CMAKE_BINARY_DIR}/ego-bundle DESTINATION ${CMAKE_INSTALL_DATADIR})
```
we should keep this away from PATH